python/cudf/cudf/tests/test_struct.py

```diff
@@ -322,3 +322,10 @@ def test_struct_int_values():
     assert isinstance(actual_series[0]["b"], int)
     assert isinstance(actual_series[1]["b"], type(None))
     assert isinstance(actual_series[2]["b"], int)
+
+
+def test_nested_struct_from_pandas_empty():
+    pdf = pd.Series([[{"c": {"x": None}}], [{"c": None}]])
```

Comment:
Does the test really matters to wrap the struct with an extra level of list? Per this PR, it looks like the minimum test coverage is just a series with 2 struct rows.

Reply:
Hmm weirdly, I can't write the C++ reproducer in Python, because there's no way to construct a Series with a parent null mask in the same way you can in C++.

This is still the simplest repro I could come up with in Python unfortunately :(